pkg/apis/apps/types.go

```diff
@@ -239,6 +255,13 @@ type StatefulSetSpec struct {
 	// increments the index by one for each additional replica requested.
 	// +optional
 	Ordinals *StatefulSetOrdinals
+
+	// volumeClaimUpdatePolicy indicates when PersistentVolumeClaims should be
```

Comment:
nit: VolumeClaimUpdatePolicy

Reply:
I'm not sure. This is not consistent. e.g. `ordinals`, `revisionHistoryLimit` above also starts with lower case.